src/components/CardView/CardView.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+
+const CardView = (props) => {
+  const { title, text, imageUrl, imageAlt, buttonUrl, buttonText } = props;
+
+  let imageEle;
+  if (imageUrl) {
+    imageEle = <img className="card-img-top" src={imageUrl} alt={imageAlt} />;
+  }
+
+  let buttonEle;
+  if (buttonUrl && buttonText) {
+    buttonEle = <a href={buttonUrl} className="btn btn-secondary" target="_blank">{buttonText}</a>;
```

Comment:
Added the noopener, but I think it's best to leave the button as-is.

If we were building out a full app here, I'd consider it, but most of this code is intended to be deleted when the app is cloned. It is an example of building out features, but once you've built your own, I expect anything we were here functionality-wise to be removed, so keen to keep that as small as possible.